index.d.ts

```diff
@@ -1700,15 +1697,15 @@ interface TableEditableCellProps extends Omit<TextTableCellProps, 'placeholder'
 export interface TableHeaderCellProps extends TableCellProps {
 }
 
-export class TableHeaderCell extends React.PureComponent<TableHeaderCellProps> {
+export class TableHeaderCell extends React.FC<TableHeaderCellProps> {
 }
 
 export interface TableHeadProps extends PaneProps {
   height?: number | string
   accountForScrollbar?: boolean
 }
 
-export class TableHead extends React.PureComponent<TableHeadProps> {
+export class TableHead extends React.FC<TableHeadProps> {
```

Comment:
If any of these are using memo or forwardRef you should use `ForwardRefComponent` custom type.

These aren't classes, so you'll want something like this:
https://github.com/segmentio/evergreen/blob/v5/index.d.ts#L1118

Reply:
If you really just want React.FC you dont need `class Foo extends`